docs/setup.md

```diff
@@ -21,24 +21,19 @@ ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/
 
 ### 3. Install python, node.js, and npm
 
-Python may be required for working with some projects.  Installing node.js will provide you with the `node` and `npm` commands.
+Python may be required for working with some projects, though OSX has it built in. Installing node.js will provide you with the `node` and `npm` commands.
 
 ```
-brew install python nodejs;
-sudo easy_install pip;
-pip install pyobjc;
-npm install -g node-gyp;
+brew install nvm;
```

Comment:
Peter, all of your proposed changes in this README look terrific to me! Except for this one line.

`nvm` discourages installation via homebrew. Per https://github.com/creationix/nvm

> Homebrew installation is not supported. If you have issues with homebrew-installed nvm, please brew uninstall it, and install it using the instructions below, before filing an issue.

For our docs, I recommend that we point people to https://github.com/creationix/nvm#install-script and say "follow these instructions".

I generally hate recommending "curl -o <url> | bash" to install anything (which is what nvm actually recommends), but at least it's not "sudo bash".